server/src/main/java/io/druid/server/coordinator/CoordinatorDynamicConfig.java

```diff
@@ -43,6 +43,7 @@
   private final boolean emitBalancingStats;
   private final boolean killAllDataSources;
   private final Set<String> killDataSourceWhitelist;
+  private final int maxSegmentsInQueue;
```

Comment:
Please document this property in code, e. g. it's not clear what queue? per-node queue, global queue? Add refs. Consider renaming to `maxSegmentsInNodeQueue` or something like that.